app/apollo/resolvers/channel.js

```diff
@@ -271,7 +271,29 @@ const channelResolvers = {
         version_uuid: versionObj.uuid,
       };
     },
-
+    uploadChannelVersion: async (parent, { org_id, uuid, content }, context)=>{
```

Comment:
we need to support S3. see here: https://github.com/razee-io/Razeedash-api/blob/master/app/apollo/resolvers/channel.js#L212 for details.